app/cfg/config.yaml

```
@@ -108,6 +109,67 @@ kubernetes_jobs:
         subPath: 'weights'
         claimName: 'mmli-clean-job-weights'
 
+  # Config for running OpenEnzymeDB job(s)
+  oed-dlkcat:
+    image: 'moleculemaker/oed-dlkcat-dla'
+    imagePullPolicy: 'Always'
+    volumes:
+      - name: 'shared-storage'
+        mountPath: '/uws/jobs/oed-dlkcat/JOB_ID/in'
+        subPath: 'uws/jobs/oed-dlkcat/JOB_ID/in'
+        claimName: 'mmli-clean-job-weights'
+      - name: 'shared-storage'
+        mountPath: '/uws/jobs/oed-dlkcat/JOB_ID/out'
+        subPath: 'uws/jobs/oed-dlkcat/JOB_ID/out'
+        claimName: 'mmli-clean-job-weights'
+  oed-unikp:
+    image: 'moleculemaker/oed-unikp'
+    imagePullPolicy: 'Always'
+    volumes:
+      - name: 'shared-storage'
+        mountPath: '/uws/jobs/oed-unikp/JOB_ID/in'
+        subPath: 'uws/jobs/oed-unikp/JOB_ID/in'
+        claimName: 'mmli-clean-job-weights'
+      - name: 'shared-storage'
+        mountPath: '/uws/jobs/oed-unikp/JOB_ID/out'
+        subPath: 'uws/jobs/oed-unikp/JOB_ID/out'
+        claimName: 'mmli-clean-job-weights'
+      - mountPath: /root/.cache/huggingface/hub
+        name: shared-storage
+        subPath: uws/jobs/oed-unikp/.cache
+        claimName: 'mmli-clean-job-weights'
+      - mountPath: /usr/app/UniKP
+        name: shared-storage
+        subPath: uws/jobs/oed-unikp/.models
+        claimName: 'mmli-clean-job-weights'
+    secrets:
+      - mountPath: '/root/.cache/token'
+        name: 'token'
+        subPath: 'token'
+        readOnly: 'true'
+        secretName: huggingface-token
+        # "itemList" takes the place of "items"
+        # This should avoid overlap with the built-in items() function
+        itemList:
+          - key: token
+            path: token
+  oed-catpred:
+    image: 'moleculemaker/oed-catpred'
+    imagePullPolicy: 'Always'
+    volumes:
+      - name: 'shared-storage'
+        mountPath: '/uws/jobs/oed-catpred/JOB_ID/in'
+        subPath: 'uws/jobs/oed-catpred/JOB_ID/in'
+        claimName: 'mmli-clean-job-weights'
+      - name: 'shared-storage'
+        mountPath: '/uws/jobs/oed-catpred/JOB_ID/out'
+        subPath: 'uws/jobs/oed-catpred/JOB_ID/out'
+        claimName: 'mmli-clean-job-weights'
+      - mountPath: /data
+        name: shared-storage
+        subPath: uws/jobs/oed-catpred/.pretrained
+        claimName: 'mmli-clean-job-weights'
+
```

Mount these models in a 2nd location to fix the missing "checkpoint" / "pt file" errors in 1.0.1
```suggestion
        claimName: 'mmli-clean-job-weights'
      - mountPath: /catpred_pipeline/catpred/production_models
        name: shared-storage
        subPath: uws/jobs/oed-catpred/.pretrained/pretrained/production
        claimName: 'mmli-clean-job-weights'
```